.env.example

```diff
@@ -1,2 +1,5 @@
-INFURA_API_KEY=zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
 MNEMONIC=here is where your twelve words mnemonic should be put my friend
+ADMIN=0x0000000000000000000000000000000000000000
```

Comment:
In the latest AccessControl PR, ADMIN address was updated to 
`0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775`

Reply:
`0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775` is not a valid Ethereum address?